# HG changeset patch
# User Andreas Weber <andreas.weber@hs-offenburg.de>
# Date 1406814561 -7200
#      Thu Jul 31 15:49:21 2014 +0200
# Node ID 19cb2530c16bce44cda6b3ccdb63517a0c3ce9d1
# Parent  f084aab05cd915999de8aeb4c12059f3dff67db5
Add __get_funcname__.m, pan.m and rotate3d.m to module.mk

diff --git a/scripts/gui/module.mk b/scripts/gui/module.mk
--- a/scripts/gui/module.mk
+++ b/scripts/gui/module.mk
@@ -1,13 +1,14 @@
 FCN_FILE_DIRS += gui
 
 gui_PRIVATE_FCN_FILES = \
   gui/private/__file_filter__.m \
   gui/private/__fltk_file_filter__.m \
+  gui/private/__get_funcname__.m \
   gui/private/__is_function__.m \
   gui/private/__uigetdir_fltk__.m \
   gui/private/__uigetfile_fltk__.m \
   gui/private/__uiobject_split_args__.m \
   gui/private/__uiputfile_fltk__.m \
   gui/private/message_dialog.m
 
 gui_FCN_FILES = \
diff --git a/scripts/plot/util/module.mk b/scripts/plot/util/module.mk
--- a/scripts/plot/util/module.mk
+++ b/scripts/plot/util/module.mk
@@ -58,22 +58,24 @@ plot_util_FCN_FILES = \
   plot/util/isprop.m \
   plot/util/linkaxes.m \
   plot/util/linkprop.m \
   plot/util/meshgrid.m \
   plot/util/ndgrid.m \
   plot/util/newplot.m \
   plot/util/__next_line_color__.m \
   plot/util/__next_line_style__.m \
+  plot/util/pan.m \
   plot/util/__plt_get_axis_arg__.m \
   plot/util/__pltopt__.m \
   plot/util/printd.m \
   plot/util/print.m \
   plot/util/refreshdata.m \
   plot/util/refresh.m \
+  plot/util/rotate3d.m \
   plot/util/saveas.m \
   plot/util/shg.m \
   plot/util/struct2hdl.m \
   plot/util/subplot.m \
   $(plot_util_PRIVATE_FCN_FILES)
 
 FCN_FILES += $(plot_util_FCN_FILES)
 
